replication/row_event.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/siddontang/go/hack"
 )
 
+var DecodeTimestamp2AsTime bool = false
```

Comment:
Maybe we should use a configuration instead of the global variable later. /cc @qiuyesuifeng